### DISCUSSION TITLE
Is it possible to delete a directory with minizip_erase?

### DISCUSSION THREAD

**Discussion by mumin16** [nmoinvaz/minizip#631] [Q&A] [answered]
Is it possible to delete directory with `minizip_erase` in zipped file?

**Comment by nmoinvaz** [comment, marked as answer]
https://github.com/zlib-ng/minizip-ng/blob/47b8449fec2c7a575a506ecbe6399ad0603b5992/minizip.c#L400

`args` contains array of strings that can contain wildcards. So you can specify `data/*` to delete all from directory.
